OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/LoginPrefs.java

```diff
@@ -39,7 +39,7 @@ public LoginPrefs(@NonNull Context context) {
 
     public void storeAuthTokenResponse(@NonNull AuthResponse response, @NonNull AuthBackend backend) {
         pref.put(PrefManager.Key.AUTH_JSON, gson.toJson(response));
-        pref.put(PrefManager.Key.SEGMENT_KEY_BACKEND, segmentKeyFromAuthBackend(backend));
+        pref.put(PrefManager.Key.SEGMENT_KEY_BACKEND, eventKeyFromAuthBackend(backend));
```

Comment:
Maybe also change the name of the constant from `SEGMENT_KEY_BACKEND` to `ANALYTICS_KEY_BACKEND`? The actual key doesn't need to be changed since that would remove backwards-compatibility, but it might make things clearer.